matRad/gui/widgets/matRad_importDicomWidget.m

```diff
@@ -993,14 +895,15 @@
     end    
     
     methods (Access = private)
-        % SCAN FUNKTION
+    % SCAN FUNKTION
```

Comment:
can we rename this to a more descriptive name for the funtion and not just "scan" ? 
like "scanFolder"  or something

Reply:
renamed here [5dcd737](https://github.com/e0404/matRad/pull/758/commits/5dcd7378230d0a3fe9d0e7d6bbb2fbfd5e386e43)